client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -57,7 +57,9 @@ class ActivityLogItem extends Component {
 				<ActivityActor { ...{ actorActivityUrl, actorName, actorRole, actorType } } />
 				<div className="activity-log-item__description">
 					<div className="activity-log-item__description-content">
-						<FormattedBlock content={ activityDescription[ 0 ] } />
+						{ activityDescription.map( ( part, i ) => (
+							<FormattedBlock key={ i } content={ part } />
```

Comment:
I think we should add a prefix or postfix to the key here, react won't render anything with colliding keys.

Reply:
there's no good way to do that here. I think in this case the index is sufficient because it's very unlikely we'd have these change within an activity and also we don't have a good way to generate a key without doing complicated logic.

in 686d2f5786 I added a comment

Reply:
you can get rid of the comment, I misremembered the key details. We should be okay here using index as long as the list isn't mutated.

Reply:
I'll leave it in since we generally discourage using indices as key :smile: